packages/fast-usdc-deploy/test/fast-usdc.test.ts

```diff
@@ -114,12 +120,45 @@ test.before('bootstrap', async t => {
     await eventLoopIteration();
   };
 
+  /**
+   * Helper to submit evidence by wallet invoke message from all configured oracles.
+   * @param evidence CCTP evidence
+   * @param invocationIdPrefix Prefix for the invocation ID
+   * @param extraArgs Additional arguments for the submitEvidence call
+   */
+  const invokeAttest = async (
+    evidence: CctpTxEvidence,
+    invocationIdPrefix: string,
+    ...extraArgs: Passable[]
+  ) => {
+    const { walletFactoryDriver: wfd } = t.context;
+    const oracles = await Promise.all(
+      oracleAddrs.map(addr => wfd.provideSmartWallet(addr)),
+    );
+
+    await Promise.all(
+      oracles.map((wallet, i) =>
+        wallet.invokeEntry({
+          id: `${invocationIdPrefix}-${i}`,
+          targetName: 'fastUsdcOperator',
+          method: 'submitEvidence',
+          args: [
+            // @ts-expect-error Type 'CctpTxEvidence' is not assignable to type 'Passable'.
+            evidence,
+            ...extraArgs,
+          ],
```

Comment:
I struggle every time I see this open-coded `{ method, args }` pattern.

Maybe something like:

```
type MessageTo<
  M extends string,
  T extends Record<M, (...args: any[]) => any>,
> = {
  id: string;
  target: string;
  method: M;
  args: Parameters<T[M]>;
};

const submitIt: MessageTo<'submitEvidence', OperatorKit['operator']> = {
          id: `${invocationIdPrefix}-${i}`,
          targetName: 'fastUsdcOperator',
          method: 'submitEvidence',
          args: [
            evidence,
            ...extraArgs,
          ]
};

wallet.invokeEntry(submitIt);
```

Reply:
I consider this change out of scope for this specific PR (especially since `attest` already follows the same pattern), but I could see this approach as a helper provided by the wallet factory helper.